cmake/compiler/compiler_flags_template.cmake

```diff
@@ -115,3 +115,5 @@ set_compiler_property(PROPERTY warning_no_pointer_arithmetic)
 
 # Compiler flags for disabling position independent code / executable
 set_compiler_property(PROPERTY no_position_independent)
+
+set_compiler_property(PROPERTY no_global_merge)
```

Comment:
Could do with a comment here, like above has, giving a simple description of what this is

Reply:
done !